products/rtpengine.md

```diff
@@ -0,0 +1,57 @@
+---
+title: rtpengine 
+category: server-app
+permalink: /rtpengine
+versionCommand: rtpengine -v
+releaseDateColumn: true
+
+identifiers:
+-   repology: rtpengine
+
+auto:
+  methods:
+    - git: https://github.com/sipwise/rtpengine.git
+      regex: '^mr(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<tiny>\d+))?$'
+
+releases:
+-   releaseCycle: "mr13.1"
```

Comment:
```suggestion
-   releaseCycle: "13.1"
    releaseLabel: "mr13.1"
```

I fear it will break our automation if we keep the "mr" in `releaseCycle`. This way users will still see `mrX.Y` and I am sure automation will work.